src/Interpreter/Runner.hs

```diff
@@ -0,0 +1,51 @@
+module Interpreter.Runner where
+
+import           Data.Map.Strict (Map)
+import           Data.Maybe (fromMaybe)
+import           Data.List (find)
+import qualified Data.Map.Strict as Map
+import           Control.Monad (forM)
+import           Control.Monad.State.Class
+import           Control.Monad.Loops
+import           Control.Monad.State.Strict
+import           Control.Monad.Cont
+import           Data.Maybe (fromMaybe, fromJust)
+import           Data.List (intercalate)
+
+import qualified Language.Mulang as Mu
+import Language.Mulang.Parsers.JavaScript (parseJavaScript)
+import Interpreter.Mulang
+import Interpreter.Mulang.Tests
+
+
+data TestResult = Success
+                | Failure String
+                deriving (Show, Eq)
+
+runTestsForDir :: String -> IO ()
```

Comment:
We should remove this function, or parameterize it, so that is not `.js`-files-specific.